packages/frontend/src/components/accounts/create/verify_account/VerifyWithPhoneOption.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import PhoneInput, { isValidPhoneNumber } from 'react-phone-number-input';
+
+import PhoneNumberInput from './PhoneNumberInput';
+import VerifyOption from './VerifyOption';
+
+export default ({
+    translateIdTitle,
+    translateIdDesc,
+    onClick,
+    activeVerificationOption,
+    disabled,
+    error,
+    onChange,
+    onBlur,
+    value
+}) => {
+
+    return (
+        <VerifyOption
+            onClick={onClick}
+            option='phone'
+            isActive={activeVerificationOption === 'phone'}
```

Comment:
Not a blocker, but it'd be nice to keep the API for these components identical to the other VerifyOption components, and keep the magic string comparisons for passing in the correct value for `isActive` in the parent component.